src/main/java/nextstep/subway/dto/PathResponse.java

```diff
@@ -4,17 +4,25 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class PathResponse {
-    private List<Station> stations = new ArrayList<>();
+    private List<StationResponse> stations = new ArrayList<>();
     private Long distance;
 
-    public PathResponse(List<Station> stations, Long distance) {
+    private PathResponse(List<StationResponse> stations, Long distance) {
         this.stations = stations;
         this.distance = distance;
     }
 
-    public List<Station> getStations() {
+    public static PathResponse from(List<Station> stations, Long distance) {
+        List<StationResponse> collect = stations.stream()
+                .map(station -> new StationResponse(station.getId(), station.getName()))
```

Comment:
DTO에서 Entity를 의존하는건 큰 문제가 되지 않으니 다음과 같이 리팩토링해볼수도 있을 것 같네요. 
```suggestion
                .map(StationResponse::from)
```

그럼 차후 Station과 StationResponse의 내부 필드가 추가되거나 하더라도 PathResponse까지 와서 매개변수를 수정하고 할 필요가 없어질 수 있겠죠.